service/src/main/java/uk/gov/hmcts/reform/fpl/events/NewHearingsAddedEvent.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.fpl.events;
+
+import lombok.EqualsAndHashCode;
+import lombok.Generated;
+import lombok.Value;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+
+
+@Value
+@Generated
+@EqualsAndHashCode(callSuper = true)
+public class NewHearingsAddedEvent extends CallbackEvent {
```

Comment:
As mentioned somewhere else I don't like this to have "Event" in the name, past tense + package already suggest that.